### DISCUSSION TITLE
Mistype URL on vs.code Docker

### DISCUSSION THREAD

**Discussion by bugalwena** [microsoft/vscode-docker#3856] [Q&A] [answered]
How do I edit a mistype in a URL for a local host on Docker vs. code? I contacted Docker support, who told me that "a functionality specifically in VS Code." could fix the cause. How or where can I edit the URL? It should be https://localhost:8080 but is missing a colon htttps//localhost:8080

The correct URL will solve many issues.

**Comment by bwateratmsft** [comment, marked as answer]
Converted to a discussion since that seemed to fit better.

The easiest way is to simply disconnect the registry and start over with adding it:
![image](https://user-images.githubusercontent.com/36966225/224068252-4eac7466-47e6-41d6-9c88-6cf9da282be2.png)

In certain circumstances we also show a "Reconnect" option, but IIRC that's only if the registry exists, but gives a 401 unauthorized response. This is intended more for facilitating password changes.
